src/types/types.ts

```diff
@@ -68,6 +68,12 @@ export type Crop = {
   cropMarker?: string
 }
 
+export const enum OptionalWords {
+  NONE = 'none',
+  LAST = 'last',
+  ANY = 'any',
```

Comment:
Why do we have just 3 options here since here https://github.com/meilisearch/milli/pull/598/files#diff-dcec10e5add7499e867436ac0283c0a108580cca33d67466eb41f85f26e841e8R292-R305 we have many others? Is this something expected?

Reply:
They are not official, @meilisearch/product-team prefer not advocating for them to soon